nixos/modules/services/monitoring/prometheus/default.nix

```diff
@@ -1739,6 +1739,17 @@ in
       '';
```

Comment:
I think people should be told about the option to only check syntax before they turn off checkConfig altogether. The section above this should probably mention chehckConfigSyntaxOnly, or perhaps checkConfig should become either a bool or "syntax-only".

Reply:
I went the latter route and made `checkConfig` become `either bool (enum ["syntax-only"])`.